src/chainbin/nnet3-chain-train.cc

```diff
@@ -53,6 +52,9 @@ int main(int argc, char *argv[]) {
                 "yes|no|optional|wait, only has effect if compiled with CUDA");
 
     opts.Register(&po);
+#if HAVE_CUDA==1
+    CuDevice::RegisterDeviceOptions(&po);
```

Comment:
If your changes require configuration to be effective, I think I would rather you set the defaults to be fast.

Reply:
Okay, the multithreaded flag is set to `False` currently, will change that to `True` but do you also want the tensor core flag default to be `True` since this has an impact on numerical roundoff?

Reply:
see my comment above